development/style-guide.md

```diff
@@ -145,60 +139,92 @@ function bank_Account() {
 }
 ```
 
-## Use UPPERCASE for Constants
+## Use `const` and `let`
+
+There is no longer a good reason to use `var`. Use `const` whenever you can,
+and `let` when you must. Hardcoded constants should be named in all UPPERCASE.
+
+*Right:*
+
+```js
+const DELAY = 10 * 1000; // ten seconds
```

Comment:
Not related to the use of const and let, I don't think the ten seconds comment is good style, as it's just saying what and not why.